msteams-platform/sbs-api-based-message-extensions.yml

```diff
@@ -495,8 +496,8 @@ items:
             "full": "Search ME API full"
         },
         "description": {
-            "short": "product app for testing API MEs",
-            "full": "product app for testing API MEs"
+            "short": "product app for testing API message extensions",
```

Comment:
Can we please align the naming to "API based message extension" across the board?

Reply:
Updated the code as suggested